### DISCUSSION TITLE
Demo 1

### DISCUSSION THREAD

**Discussion by JohanCSmit** [COS301-SE-2022/Exploring-Self-Sovereign-Identity#10] [General]
Discussion for any work related to Demo 1.

**Comment by JohanCSmit** [comment]
@MxolisiMkanzi we need to setup our environment so we can start working by tomorrow. Would you be able to assist me with this?

**Comment by RebeccaPillay** [comment]
Initial SSI System Requirements: To be reviewed and discussed in client meeting. 
@JohanCSmit @H-e-n-r-i 

[CodeOfDuty - SSI - Initial Requirements Identification .pdf](https://github.com/COS301-SE-2022/Exploring-Self-Sovereign-Identity/files/8619563/CodeOfDuty.-.SSI.-.Initial.Requirements.Identification.pdf)

**Reply by JohanCSmit** [reply]
Could we possibly adjust it to the following:

R1 The Derivco-Code of Duty SSI System shall have User Accounts
             R1.1 Users shall be able to claim ownership over personal data and identities (digital assets)
             etc.
R2 The Derivco-Code of Duty SSI System shall have SSIs (digital identities)
             R2.1  SSIs shall be linked to real world data
             etc.

**Reply by JohanCSmit** [reply]
Here are some Highlights from the user story (original document's description) which you could use:
[image](https://user-images.githubusercontent.com/87640748/166683205-410addf4-3de7-4735-9f7d-2e946c457a2b.png)

**Reply by RebeccaPillay** [reply]
See adjusted document, with reference to your notes. @JohanCSmit 

[CodeOfDuty - SSI - Initial Requirements Identification 2.0 .pdf](https://github.com/COS301-SE-2022/Exploring-Self-Sovereign-Identity/files/8621203/CodeOfDuty.-.SSI.-.Initial.Requirements.Identification.2.0.pdf)

**Reply by JohanCSmit** [reply]
Thanks that looks great! @RebeccaPillay

**Reply by JohanCSmit** [reply]
I believe after our client meeting we'll be able to add a lot more and expand as well. @RebeccaPillay

**Comment by RebeccaPillay** [comment]
Basic SSI System Architecture: To be reviewed and discussed in client meeting. 
@JohanCSmit @MxolisiMkanzi 

[Code of Duty - SSI - Basic Architecture .pdf](https://github.com/COS301-SE-2022/Exploring-Self-Sovereign-Identity/files/8619578/Code.of.Duty.-.SSI.-.Basic.Architecture.pdf)

**Reply by JohanCSmit** [reply]
This looks good! If @MxolisiMkanzi agrees we can start working on a formal architecture.
Schalk did mention that they have an architecture team so we might just need to see what happens.

**Comment by JohanCSmit** [comment]
@10416260 Possible Use Cases:
[User-Manage](https://user-images.githubusercontent.com/87640748/166723212-38be86b6-6a9c-4714-9641-cc795263f0c9.jpg)
[User-Issuer](https://user-images.githubusercontent.com/87640748/166711366-af719525-5c73-4b6d-9f6e-d74028974b59.jpg)

**Comment by JohanCSmit** [comment]
@10416260 @MxolisiMkanzi 

I have finished the 'connect' endpoint, please review it if you'd like and notify me of any issues/concerns when you are able.
